### PR TITLE
Fix test upload workflow

### PR DESCRIPTION
**Pull request by sophiewigmore**
## Summary

The addition of license metadata has caused some breakages in our workflow. The license metadata is an array of strings, and in the `build-upload` workflow, it broke because the quotes around the variable messed up the parsing. See [these failed workflows](https://github.com/paketo-buildpacks/dep-server/runs/3059477161?check_suite_focus=true) for examples of this. 

Additonally, the presence of the environment variables in the `test-upload` workflow were only there so we could run a manual workflow dispatch. This isn't necessary, since so many fields are manually needed to enter it wasn't used often. Removing this should alleviate difficulties passing in the array of license metadata we see when we tried to send a dispatch in [this workflow.](https://github.com/paketo-buildpacks/dep-server/runs/3060280377?check_suite_focus=true).

TLDR; these changes should make it so we can pass in an array of licenses succesfully, rather than having to convert the array into a single string.

## Checklist

* [X] I have viewed, signed, and submitted the Contributor License Agreement.
* [ ] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [X] I have added an integration test, if necessary.
* [X] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [X] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

File: scripts/update-all-metadata/go.mod [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module example.com/m
```
Comment (brayanhenao) [inline scripts/update-all-metadata/go.mod:1]: It's not a good module name, what about `github.com/paketo-buildpacks/dep-server/scripts/update-all-metadata`

File: scripts/update-all-metadata/main.go [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+		dispatch.EventType = "tini-test"
+		dispatch.ClientPayload = json.RawMessage(payload)
+
+		payloadData, err := json.Marshal(&dispatch)
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		fmt.Println(string(payloadData))
+		// // send dispatch w all the info for each version
```
Comment (brayanhenao) [inline scripts/update-all-metadata/main.go:81]: Do we need this commented code or can it be deleted?

**Review by brayanhenao — APPROVED** [review]
